connectors/sources/sharepoint.py

```diff
@@ -774,8 +877,14 @@ async def get_docs(self, filtering=None):
         for site_url in server_relative_url:
             async for list_data in self.sharepoint_client.get_lists(site_url=site_url):
                 for result in list_data:
+                    is_site_page = False
+                    selected_field = "Modified,Id,GUID,File,Folder"
```

Comment:
This might be just 2 constants right? 

```
CLOUD_SELECTED_FIELDS = "Modified,Id,GUID,File,Folder"
SERVER_SELECTED_FIELDS ="WikiField, Modified,Id,GUID,File,Folder" 
```